docs/API.md

```diff
@@ -215,6 +215,31 @@ If the name is not provided, the task will be named after the function
 Since the task can be run from the command line, you should avoid using
 spaces in task names.
 
+#### fn
+
+The function that performs the task's operations. Generally it takes this form:
+```
+gulp.task('somename', function() {
```

Comment:
make sure to use named functions here because we are discouraging anonymous functions + string names.